src/emailLogin/README.md

```diff
@@ -0,0 +1,44 @@
+# Email migration script
+
+## How to use
+
+### Setup
+
+The following steps are needed before you can run the email login migration.
+
+First, ensure that you are connected to [AWS VPN](https://www.notion.so/opengov/Instructions-to-use-OGP-s-AWS-VPN-e67226703cac459999b84c02200a3940) as only the VPN is whitelisted to use the EC2 instance<sup>1</sup>.
+
+Next, you will require the correct environment variables and credentials.
+
+- Go into the 1PW Isomer - Admin vault and search for the `.ssh/.env.<staging | prod>` file.
+- Create a folder named .ssh in the root directory and place the `.env` files there.
+- Search for the corresponding credentials `isomercms-<staging | production>-bastion.pem`
+- Put these credentials into the .ssh folder also.
+
+Also, ensure that the repo name and the team name of the github repository are the same before starting.
+
+### Running the migration
```

Comment:
checking in here, this isnt intended right, since this was the previous migration instruction? 
I am ok with having mutliple README, but can I get your help to just link this from the main project root's readme?

Reply:
This is still needed! The first part is setup and where to retrieve credentials, this part is actually running it